presto-base-jdbc/src/main/java/io/prestosql/plugin/jdbc/BaseJdbcConfig.java

```diff
@@ -47,6 +51,22 @@ public BaseJdbcConfig setConnectionUrl(String connectionUrl)
         return this;
     }
 
+    public Set<String> getJdbcTypesMappedToVarchar()
+    {
+        return jdbcTypesMappedToVarchar;
+    }
+
+    @Config("jdbc-types-mapped-to-varchar")
+    public BaseJdbcConfig setJdbcTypesMappedToVarchar(String jdbcTypesMappedToVarchar)
+    {
+        if (jdbcTypesMappedToVarchar != null) {
+            for (String type : jdbcTypesMappedToVarchar.split(",")) {
+                this.jdbcTypesMappedToVarchar.add(type.trim());
```

Comment:
- You should reset `jdbcTypesMappedToVarchar` when setter is called.
- avoid `String#split` for all its weirdnesses

```
this.jdbcTypesMappedToVarchar = ImmutableSet.copyOf(Splitter.on(",").omitEmptyStrings().split(nullToEmpty(jdbcTypesMappedToVarchar)));
```

Reply:
Whops.. add `.trimResults()` too.